test_outputs/enum_repr_int_removed.output.ron

```diff
@@ -38,5 +38,33 @@
         "span_filename": String("src/main.rs"),
         "span_begin_line": Uint64(16),
         "visibility_limit": String("public"),
+    },
+    {
+        "name": String("U8Enum"), 
+        "path": List([String("enum_repr_int_changed"), String("U8Enum")]), 
+        "span_begin_line": Uint64(29), 
+        "span_filename": String("src/main.rs"), 
+        "visibility_limit": String("public")
+    },
+    {
+        "name": String("I32Enum"), 
+        "path": List([String("enum_repr_int_changed"), String("I32Enum")]), 
+        "span_begin_line": Uint64(34), 
+        "span_filename": String("src/main.rs"), 
+        "visibility_limit": String("public")
+    }, 
+    {
+        "name": String("IsizeEnum"), 
+        "path": List([String("enum_repr_int_changed"), String("IsizeEnum")]), 
+        "span_begin_line": Uint64(39), 
+        "span_filename": String("src/main.rs"), 
+        "visibility_limit": String("public")
+    }, 
+    {
+        "name": String("UsizeEnum"), 
+        "path": List([String("enum_repr_int_changed"), String("UsizeEnum")]), 
+        "span_begin_line": Uint64(44), 
+        "span_filename": String("src/main.rs"), 
+        "visibility_limit": String("public")
```

Comment:
This doesn't make any sense whatsoever. The `span_begin_line` in those outputs don't exist in the `new/src/main.rs` (the file has only 19 lines). Do you have any idea what happened here @obi1kenobi ?

Reply:
I'm guessing the span begin line numbers are the old numbers from before you refactored the test structure. Note that this test is currently failing due to a mismatch between the actual and expected test data. You may need to just update the line numbers to match the new file (which doesn't have feature-flagged code so of course the line numbers don't match up).

Reply:
Still, something weird happened here. This test output I've generated just now by running the script and then `cargo test`. I'm sure that this test output is not a mistake, because it has been generated just like the other two test outputs that have been failing before the latest commit. And the CI have just been completed for this commit and it is now passing.

Reply:
Ah, now that the code uses `map` instead of `flatten`, I immediately see that they are from another crate, not from this same crate. The output seems to be ok.